src/components/return/ReturnTable.js

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import {
+  Checkbox,
+  makeStyles,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+  TableSortLabel,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+
+  return 0;
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
```

Comment:
+1/-1 changes here, only. Maybe this way would be easier to read and understand?
`return (a,b) => descendingComparator(a, b, orderBy) * (order === 'desc' ? 1 : -1)`